utils/capellacontrol/controller.go

```diff
@@ -555,6 +555,53 @@ type CreateClusterRequest_Spec struct {
 	Services        []string                              `json:"services"`
 }
 
+type CreateDeployClusterRequest struct {
+	CIDR        string                           `json:"cidr"`
+	Description string                           `json:"description"`
+	Name        string                           `json:"name"`
+	Package     string                           `json:"package"`
+	TenantId    string                           `json:"tenantId"`
+	ProjectId   string                           `json:"projectId"`
+	Provider    string                           `json:"provider"`
+	Region      string                           `json:"region"`
+	Server      string                           `json:"server"`
+	Override    CreateOverrideRequest            `json:"overRide,omitempty"`
+	SingleAZ    bool                             `json:"singleAZ"`
+	Specs       []CreateDeployClusterRequestSpec `json:"specs"`
+	Timezone    string                           `json:"supportTimezone"`
+}
+
+type CreateDeployClusterRequestSpec struct {
```

Comment:
Following the same code-style as the rest of cbdinocluster, this should be `DeployClusterRequest_Spec`